### ISSUE TITLE
update `autocfg` dependecy version, for no conflicts

### ISSUE THREAD

**Issue by damooo** [svartalf/rust-claim#9]
Currently, `autocfg` crate dependecy set to `~1.0`, and pegged to minor release.

It's current active minor version is `1.1`, and many crates like `dashmap`, etc. use that version.

And this causes conflicts, and render claim unusable with many crates.

It should better pegged to major version, like `^1`.

**Comment by Anders429** [comment]
I'm being affected by this quite a bit. This crate is almost unusable in any larger project due to the versioning conflict. Turns out `autocfg` is a really popular dependency lol.

For reference, [this `cargo` issue](https://github.com/rust-lang/cargo/issues/5340) calls out the tilde operator as violating version contract. I think removing it, as is done in #10, is definitely the correct solution.

**Comment by Turbo87** [comment]
FWIW you can work around it like in https://github.com/rust-lang/crates.io/pull/4857/files, but it's obviously not a long-term solution

**Comment by dmrolfs** [comment]
agreed this is an issue and claim is problematic to use without this update. This is a really useful test crate. Hopefully it'll be added and published!

**Comment by ghost** [comment]
I forked this library and created https://github.com/mattwilkinsonn/rust-claims so I can continue to use this library. It is available on crates.io - https://crates.io/crates/claims. Feel free to use this, at least until this library can be updated again.

**Comment by Anders429** [comment]
@mattwilkinsonn thanks a ton! Do you have any interest in also merging #8 into your fork?

**Comment by yanns** [comment]
Thanks a lot for the temporary fork. This has unblocked us for dependencies update!

**Comment by jclmnop** [comment]
> I forked this library and created https://github.com/mattwilkinsonn/rust-claims so I can continue to use this library. It is available on crates.io - https://crates.io/crates/claims. Feel free to use this, at least until this library can be updated again.

Thanks, I was starting to get a massive headache from this.
